eclass/wxwidgets.eclass

```diff
@@ -116,11 +116,11 @@ setup-wxwidgets() {
 	fi
 
 	wxconf="${wxtoolkit}-unicode-${wxdebug}${WX_GTK_VER}"
+	for wxconf in "${CHOST:-${CBUILD}}-${wxconf}" "${wxconf}"; do
+		[[ -f ${ESYSROOT}/usr/$(get_libdir)/wx/config/${wxconf} ]] && break
+	done || die "Failed to find configuration ${wxconf}"
 
-	[[ ! -f ${EPREFIX}/usr/$(get_libdir)/wx/config/${wxconf} ]] \
-		&& die "Failed to find configuration ${wxconf}"
-
-	export WX_CONFIG="${EPREFIX}/usr/$(get_libdir)/wx/config/${wxconf}"
+	export WX_CONFIG="${ESYSROOT}/usr/$(get_libdir)/wx/config/${wxconf}"
```

Comment:
is this guaranteed to continue working on say, Prefix with EAPI 6?

Reply:
I thought portage had some backwards compatibility so it's defined sensibly in older EAPIs, but it's not available according to PMS.  I guess it could use `${ESYSROOT:-${EPREFIX}}`.